Create lwm2m-agent-1021-11-27-fixed-null-pointer-exception-in-composite-read-device-operation.md (#4393)
SoftwareAG/c8y-docs: content/change-logs/device-management/lwm2m-agent-1021-11-27-fixed-null-pointer-exception-in-composite-read-device-operation.md

---
date: ""

> title: Fixed NullPointerException in composite read device operation

Reviewer: ```suggestion
title: Fixed NullPointerException in composite read device operations
```

product_area: Device management & connectivity
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-1KLUzmqfe
    label: LWM2M
build_artifact:
  - value: tc-ggH2M4hf3
    label: lwm2m-agent
ticket: DM-5791
version: 1021.11.27
---

> Description: Fixed a NullPointerException that occurred when performing composite read operations on LWM2M resources or objects that do not exist on the device.

Reviewer: ```suggestion
Previously, a NullPointerException occurred when performing composite read operations on LWM2M resources or objects that did not exist on the device. This issue has been resolved.
```
